app/uk.gov.hmrc.helptosavestridefrontend/models/NSIPayload.scala

```diff
@@ -62,8 +62,20 @@ object NSIPayload {
             contactDetails: ContactDetails): NSIPayload = NSIPayload(forename, surname, dateOfBirth, nino, contactDetails, "callCentre", "STRIDE")
 
   def apply(details: ApplicantDetails, nino: NINO): NSIPayload =
-    NSIPayload(details.forename, details.surname, details.dateOfBirth, nino,
-               ContactDetails(details.address1, details.address2, details.address3,
-                              details.address4, details.address5, details.postcode, Some(details.countryCode)))
+    NSIPayload(
+      details.forename,
+      details.surname,
+      details.dateOfBirth,
+      nino,
+      ContactDetails(
+        details.address1,
+        details.address2,
+        details.address3,
+        details.address4,
+        details.address5,
+        details.postcode,
+        Some(details.countryCode)
+      )
+    )
```

Comment:
I'd prefer you didn't commit this change, it is an auto-formatting change, I assume.  But why change a file who's functionality you aren't changing, it just adds to history of the file, it seems to me.

If you are sue that this is formatting by scalafmt and not going to change again, then I can see yopu could argue for including it.

Your call